dockers/texcmp/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:15.04
+MAINTAINER Martin von Gagern <gagern@ma.tum.de>
+RUN sed -i 's/^\(deb.*updates\)/#\1/' /etc/apt/sources.list && apt-get update
```

Comment:
Disable normal updates, so we don't receive any bug fix releases or similar, just security relevant fixes. The benefit is that we don't have to pin the versions of the packages we install (which are quite numerous, with all the indirect deps pulled in), and still rely on the same versions getting installed in subsequent runs unless something drastic forces a security update to one of the packages. It's one more step towards better reproducibility.